gunicorn/config.py

```diff
@@ -465,11 +465,13 @@ class ConfigFile(Setting):
     name = "config"
     section = "Config File"
     cli = ["-c", "--config"]
-    meta = "FILE"
+    meta = "CONFIG"
     validator = validate_string
     default = None
     desc = """\
-        The path to a Gunicorn config file, or python module.
+        The Gunicorn config file.
+
+        A string of the form 'PATH', 'file:PATH', or 'python:MODULE_NAME'.
```

Comment:
We can also add a "versionadded" directive (or we can add a note to docs/source/news.rst)